src/main/java/org/sonatype/nexus/ci/config/Nxrm3Configuration.groovy

```diff
@@ -62,7 +62,7 @@ class Nxrm3Configuration
         throws IOException
     {
       try {
-        def repositories = getApplicableRepositories(serverUrl, credentialsId)
+        def repositories = getApplicableRepositories(serverUrl, credentialsId, 'maven2')
         ok("Nexus Repository Manager 3.x connection succeeded (${repositories.size()} hosted repositories)")
```

Comment:
I had changed this message to reflect the total number of hosted repositories after staging move's prior merge (due to the change in what repos were returned) alongside fixing the component upload due to the API recent changes.

Either way is fine, really, but I'd say revert this message back to '...hosted maven repositories' if we're doing the filter on line 65.